registry/storage/driver/s3-goamz/s3.go

```diff
@@ -267,8 +267,8 @@ func New(params DriverParameters) (*Driver, error) {
 	if params.V4Auth {
 		s3obj.Signature = aws.V4Signature
 	} else {
-		if params.Region.Name == "eu-central-1" {
```

Comment:
I wish to combine the else and if, like:
```
else if mustV4Auth(params.Region.Name) {
       .......
}
```
Maybe this makes code more readable.